testbed/testbed/load_generator.go

```diff
@@ -380,18 +361,8 @@ func (lg *LoadGenerator) generateMetrics() {
 
 	metricSender := lg.sender.(MetricDataSender)
 
-	// Generate up to 10 data points per metric.
-	const dataPointsPerMetric = 10
-
-	// Calculate number of metrics needed to produce require number of data points per batch.
-	metricCount := int(lg.options.ItemsPerBatch / dataPointsPerMetric)
-	if metricCount == 0 {
-		log.Fatalf("Load generator is configured incorrectly, ItemsPerBatch is %v but must be at least %v",
-			lg.options.ItemsPerBatch, dataPointsPerMetric)
-	}
-
-	// Keep count of generated data points.
-	generatedDataPoints := 0
+	// Generate 7 data points per metric.
+	const dataPointsPerMetric = 7
```

Comment:
Why do we have 7 datapoints for OTLP metric and 10 for OC?

Reply:
Was trying to sabotage :)))